src/components/RangeSlider/customTheme.scss

```diff
@@ -0,0 +1,226 @@
+@import '~@kununu/kununu-theme-v2/scss/base/variables';
+
+$height: 30px;
+$thumb-height: 25px;
+$track-height: 4px;
+
+$upper-color: lighten($red, 40);
+$lower-color: $blue;
+$thumb-color: $red;
+$thumb-hover-color: $blue;
+
+$upper-background: linear-gradient(to top, $upper-color, $upper-color) 100% 50% / 100% $track-height no-repeat transparent;
+$lower-background: linear-gradient(to top, $lower-color, $lower-color) 100% 50% / 100% $track-height no-repeat transparent;
+
+// Webkit cannot style progress so we fake it with a long shadow on the thumb element
+@function webkit-slider-thumb-shadow($i: 1) {
+  $val: #{$i}px 0 0 -#{($thumb-height - $track-height) / 2} #{$upper-color};
+  @if $i == 1 {
+    @for $k from 2 through 2000 {
+      $val: #{$val}, webkit-slider-thumb-shadow($k);
+    }
+  }
+  @return $val;
+}
+
+@mixin thumb {
+  height: $thumb-height;
+  width: $thumb-height;
+  background: #fff;
+  border-radius: 100%;
+  border: 7px solid $thumb-color;
+}
+
+@mixin hover-effect {
+  transition: border .2s ease-in-out;
+  border-color: darken($thumb-color, 10);
+}
+
+.slider {
+  appearance: none; // NOSONAR
+  -webkit-appearance: none; // NOSONAR
+  overflow: hidden;
+  cursor: pointer;
+  height: 26px;
+  margin: 7px 0 10px 0;
+
+  // fix slider height on IE11
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+    height: 74px;
+    margin-top: -8px;
+  }
+
+  &:focus {
+    position: relative;
+    outline: none;
+
+    &::-webkit-slider-thumb {
```

Comment:
would this still work, if you just comma separate the three vendor slider-thumbs? They all have the same styling

Reply:
nope, tested